autoload/minimap/vim.vim

```diff
@@ -145,12 +145,21 @@ function! s:open_window() abort
 
     augroup MinimapAutoCmds
         autocmd!
-        autocmd QuitPre *                               let g:minimap_did_quit = 1
-        autocmd WinEnter <buffer>                       call s:handle_autocmd(0)
-        autocmd WinEnter *                              call s:handle_autocmd(1)
-        autocmd BufWritePost,VimResized *               call s:handle_autocmd(2)
-        autocmd BufEnter,FileType *                     call s:handle_autocmd(3)
-        autocmd FocusGained,CursorMoved,CursorMovedI *  call s:handle_autocmd(4)
+        autocmd QuitPre *                                       let g:minimap_did_quit = 1
+        autocmd WinEnter <buffer>                               call s:handle_autocmd(0)
+        autocmd WinEnter *                                      call s:handle_autocmd(1)
+        autocmd BufWritePost,VimResized *                       call s:handle_autocmd(2)
+        autocmd BufEnter,FileType *                             call s:handle_autocmd(3)
+        autocmd FocusGained,CursorMoved,CursorMovedI <buffer>   call s:handle_autocmd(4)
+        if g:minimap_highlight_range == 1
+            " Vim and Neovim (pre-November 2020) do not have a WinScrolled autocmd event.
+            if g:minimap_win_scrolled_exists == 1
+                autocmd FocusGained,WinScrolled *               call s:handle_autocmd(5)
+            else
+                autocmd FocusGained,CursorMoved,CursorMovedI *  call s:handle_autocmd(5)
```

Comment:
Hi @rabirabirara I find `exists('##WinScrolled')` function can test if the target event is supported by vim/neovim. We may use it to replace the `g:minimap_win_scrolled_exists` option?

Reply:
I am embarrassed... I tunnel-visioned so hard on checking for a user's version that I didn't bother to look up "checking if an autocmd exists".  I removed the option, replaced it, and put the change in master.  Thanks!

Reply:
It’s okay, I happen to know that there is this way. I really like this feature! Thank for your brilliant work! @rabirabirara